src/main/java/core/basesyntax/figure/Circle.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.figure;
+
+public class Circle extends Figure {
+
```

Comment:
[Don't begin class or method implementation with an empty line.](https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#dont-begin-class-or-method-implementation-with-an-empty-line)

Reply:
not fixed